core/src/avm1/globals/shared_object.rs

```diff
@@ -253,7 +253,7 @@ fn deserialize_object_json<'gc>(
         Some(activation.context.avm1.prototypes.object),
     );
     for entry in json_obj.iter() {
```

Comment:
Let's do `for (name, value) in ...` (for all instances in this file), it's much more clean and readable.